test/base/test.ml

```diff
@@ -114,6 +114,29 @@ let _ = convert_longident ")"
 Exception: Invalid_argument "Ppxlib.Longident.parse: \")\"".
 |}]
 
+(* FIXME this is a bug *)
```

Comment:
What is the bug exactly? It would make sense to add more details here!

Reply:
Ouh my bad, I forgot to remove that comment, I added the fix to the PR!

Reply:
To answer your question, the bug discovered by the initial test addition was that `+.` was parsed as:
`Ldot (Lident "+", "")` instead of `Lident "+."`.

I added a fix for this.

In the long run we might want to make this function more robust, as was mentioned in https://github.com/ocaml-ppx/ppxlib/pull/98.